web/vtctld2/src/app/shared/flags/keyspace.flags.ts

```diff
@@ -45,6 +45,15 @@ export class RebuildKeyspaceGraphFlags {
   }
 }
 
+export class ReloadKeyspaceSchemaFlags {
```

Comment:
This must be named ReloadSchemaKeyspaceFlags to be consistent with the vtctl command.